hms-app/src/components/card-details/HackathonDetails.tsx

```diff
@@ -261,7 +261,7 @@ export default function HackathonDetails(props: IProps) {
         </div>
       )}
 
-      {hackathonData.startDate &&
```

Comment:
this looks really hacky, and it isn't clear from the code how `> new Date(1)` is a valid check :thinking:

Reply:
would be > new Date(1970-01-01T00:00:00.000Z) better? somehow i couldnt check for if title != '' or somethink like that so i thought i check for date not for one of the strings

Reply:
thats not really good either, we should work it so that if startDate is null or undefined then it works and if it ever has a value then it is valid :thinking:

Reply:
sounds like a good compromise